linux_os/guide/system/software/integrity/software-integrity/aide/aide_disable_silentreports/rule.yml

```diff
@@ -0,0 +1,49 @@
+documentation_complete: true
+
+prodtype: ubuntu2004
+
+title: 'AIDE must notify personnel if baseline configurations are altered'
```

Comment:
In alignment to [Style Guide](https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#rule-sections) it must be in [Title case](https://en.wikipedia.org/wiki/Title_case)

Reply:
Updated in https://github.com/ComplianceAsCode/content/pull/10738/commits/c7c9463317a6570a08df6710bd1d98692ea3ae11